chaincode/ledger.go

```diff
@@ -191,13 +191,21 @@ type ComputePlan struct {
 	AggregatetupleKeys      []string             `json:"aggregatetupleKeys"`
 	AssetType               AssetType            `json:"assetType"`
 	CompositeTraintupleKeys []string             `json:"compositeTraintupleKeys"`
-	DoneCount               int                  `json:"DoneCount"`
 	IDToTrainTask           map[string]TrainTask `json:"IDToTrainTask"`
-	Status                  string               `json:"status"`
+	State                   ComputePlanState     `json:"-"`
```

Comment:
Making the intent clearer can help with readability:

```suggestion
	State                   ComputePlanState     `json:"-"` // "-" means this field is excluded from JSON (de)serialization
```

Reply:
👍  Indeed that's a good idea.